rules/chrome.json

```diff
@@ -0,0 +1,55 @@
+{
+  "patterns": [
+    "\\b[cC]hrome\\b\\s+\\b[bB]rowser\\b",
+    "\\b[gG]oogle\\b\\s+\\b[cC]hrome\\b"
+  ],
+  "dependencies": [
+    {
+      "pre_install": [
+        { "command": "if [ -z $(which google-chrome) ]; then wget -O - https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb > /tmp/google-chrome.deb; fi" },
```

Comment:
For a shorter command, download it directly to file, rather than to stdout and redirected?

```sh
wget -O /tmp/google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
```

Reply:
```suggestion
        { "command": "if [ -z $(which google-chrome) ]; then wget -O /tmp/google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb; fi" },
```